packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -282,6 +284,16 @@ storage:
           kind: KubeletConfiguration
           cgroupDriver: "$${docker_cgroup_driver}"
           EOF
+    - path: /opt/etcd-private-interface
+      filesystem: root
+      mode: 0744
+      contents:
+        inline: |
+          #!/bin/bash
+          set -e
+          echo "ETCD_LISTEN_CLIENT_URLS=https://$$COREOS_PACKET_IPV4_PRIVATE_0:2379" > /etc/kubernetes/etcd.config
+          echo "ETCD_LISTEN_PEER_URLS=https://$$COREOS_PACKET_IPV4_PRIVATE_0:2380" >> /etc/kubernetes/etcd.config
+          echo "ETCD_LISTEN_METRICS_URLS=https://$$COREOS_PACKET_IPV4_PRIVATE_0:2381" >> /etc/kubernetes/etcd.config
```

Comment:
I would still put this here at the end unless you have already checked that the EnvironmentFile gets reevaluated for each Exec* step:
```
systemctl restart etcd-member &
exit 1
```

Reply:
It is not needed anymore, since it is happening in a separate service.